multiqc/modules/samtools/rmdup.py

```diff
@@ -68,14 +68,14 @@ def parse_samtools_rmdup(self):
         # General Stats Table
         stats_headers = {
             "pct_dups": {
-                "title": "% Dups",
+                "title": "Dups",
```

Comment:
Maybe we can use a few more characters here..
```suggestion
                "title": "Duplicates",
```